### DISCUSSION TITLE
git actions are not running

### DISCUSSION THREAD

**Discussion by NARENDRA935** [learndevopsonline/learndevopsonline#1820] [Q&A]
hii , raghu my git actions are not running and runners also fine but still facing the issue
<img width="895" alt="Screenshot 2024-02-19 165840" src="https://github.com/learndevopsonline/learndevopsonline/assets/153573686/98a92ad6-c51f-448d-af31-22cb90432992">
<img width="768" alt="Screenshot 2024-02-19 170123" src="https://github.com/learndevopsonline/learndevopsonline/assets/153573686/eba03504-f4db-4d52-8fd3-f424d09200d6">

**Comment by chaithanya1812** [comment]
Can you paste the code of your workflow

**Reply by NARENDRA935** [reply]
name: Developer_Branch_CI
on:
  push:
    branches:
      - 'XPNS-*'
jobs:
  check-bats-version:
    runs-on: self-hosted
    steps:
      - run: env

**Comment by chaithanya1812** [comment]
Can you join lab session now

**Reply by NARENDRA935** [reply]
okay
